apps/backoffice/src/pages/new-group-api-key.tsx

```diff
@@ -0,0 +1,171 @@
+import { ButtonCancel, ButtonNext } from "@/components/buttons";
```

Comment:
This page should be nested in "keys" page

Reply:
[fix](https://github.com/pagopa/io-services-cms/pull/1047/commits/1db82e3f28ad9a3dabb402099f2f006d3175c2e3)